cmd/codebuild/reset/main.go

```diff
@@ -221,7 +221,7 @@ func generateNukeConfig(svc *service, f io.Writer) error {
 		AdminRole:       config.accountAdminRoleName,
 		PrincipalRole:   config.accountPrincipalRoleName,
 		PrincipalPolicy: config.accountPrincipalPolicyName,
-		Regions:         config.allowedRegions,
```

Comment:
I changed `allowedRegions` to `regions`. We're not really "allowing" anything here -- it's the list of regions we want to nuke